client/galaxy/scripts/apps/analysis.js

```diff
@@ -94,15 +94,20 @@ window.app = function app(options, bootstrapped) {
             var model = new UserPreferences.Model({
                 user_id: Galaxy.params.id
             });
-            this.page.display(new FormWrapper.View(model.get(form_id)));
+            this.page.display(new FormWrapper.View(_.extend(
+                model.get(form_id),
+                {active_tab: "user"}
```

Comment:
This should probably be a parameter of `page` instead of the underlying view:
`this.page.display(VIEW, {active_tab:  "user"})`.

Reply:
@guerler Unfortunately it's not as clean in this particular instance due to the generic `FormWrapper` used, but the approach was chosen to allow better separation of concerns -- that is, allowing most concrete views to document/mark their own titles/tabs as appropriate.  I'll agree it's a little complicated in situations like the generic display above, requiring this extra context in the controller action, but as we build out more concrete views these should continue to go away.

We can revisit it later, (after an upcoming rewrite of the masthead, preferably -- something I've been poking at), but I think delegating to the toplevel view attributes as has been done here (and in previous PRs) is the best path forward now.

Reply:
Alright, sounds good.